.editorconfig

```diff
@@ -1,15 +1,14 @@
-# This file is for unifying the coding style for different editors and IDEs	
+# This file is for unifying the coding style for different editors and IDEs
 # See editorconfig.org
 
 # top-most EditorConfig file
 root = true
 
-# Set all files to utf-8
 [*]
 charset = utf-8
-
-# Enforce line feed at end of line (instead of carriage return + line feed)
-# Prettier doesn't stricly enforce it so we do it here
-indent_style = space
-indent_size = 2
 end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = false
+max_line_length = 80
```

Comment:
I think we can keep this property in the prettierrc. It's [not universally supported](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties) on all editors' plugins.

Editorconfig's feature research blog state to just use [ruler guides](https://github.com/editorconfig/editorconfig/wiki/Property-research:-maximum-line-length) on your editor for this

Reply:
Also, I think 80 is also prettier's default printWidth, my eslint used prettier rules to fix longer lines. We may not even need it